apps/gov-website/pages/api/identity/judgement.tsx

```diff
@@ -10,6 +11,19 @@ import {
 	signAndSendPromise,
 } from "@app-gov/service/cennznet";
 import { CENNZ_NETWORK } from "@app-gov/service/constants";
+import { DISCORD_BOT } from "@app-gov/service/constants";
+
+const bot = new Discord.Client({
+	partials: ["CHANNEL"],
+	intents: [
+		Discord.Intents.FLAGS.GUILDS,
+		Discord.Intents.FLAGS.GUILD_MESSAGES,
+		Discord.Intents.FLAGS.GUILD_MEMBERS,
+		Discord.Intents.FLAGS.DIRECT_MESSAGES,
+	],
+});
+
+bot.login(DISCORD_BOT.Token);
 
```

Comment:
Can we create a `service/discord` package the encapsulate this logic of getting the bot.

It should be an async method `getDiscordBot(token: string)` which resolve the `bot` when it triggers `"ready"` event.